### PR TITLE
all baskets

### PR DESCRIPTION
**Pull request by gh-oss-contributor**
checking baskets

### CODE REVIEW

File: src/shared/Items.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ function Items() {
   return (
     <div className="flex content-center flex-wrap h-48">
       <div className="w-1/4 p-2">
-        <Basket />
+        <Apple />
```
Comment (scriptnull) [inline src/shared/Items.js:8]: should be basket

File: src/shared/Items.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ function Items() {
         <Basket />
       </div>
       <div className="w-1/4 p-2">
-        <Apple />
+        <Basket />
```
Comment (scriptnull) [inline src/shared/Items.js:17]: correct

Comment (scriptnull) [inline src/shared/Items.js:17]: double correct

Comment (scriptnull) [inline src/shared/Items.js:17]: correct!

Comment (scriptnull) [inline src/shared/Items.js:17]: super correct

Comment (gh-team-member) [reply src/shared/Items.js:17]: double correct

File: src/shared/Items.js (hunk 1 above)
Comment (scriptnull) [inline src/shared/Items.js:8]: this should be basket

Comment (scriptnull) [reply src/shared/Items.js:8]: yes yes

**Comment by scriptnull** [comment]
such a  nice PR

**Pull request closed without merging**